tox.ini

```
@@ -3,17 +3,18 @@ isolated_build = true
 envlist =
     docs
     packaging
-    py{38,39,310}-dj{32,41,42}-{sqlite,postgresql,postgis,mysql}
+    py{38,39,310}-dj{32,41,42}-{sqlite,legacy_postgresql,postgis,mysql}
     py{310}-dj{40}-{sqlite}
-    py{310,311}-dj{41,42,main}-{sqlite,postgresql,postgis,mysql}
+    py{310,311}-dj{41,42,main}-{sqlite,legacy_postgresql,postgresql,postgis,mysql}
```

```suggestion
    py{310,311}-dj{41}-{sqlite,legacy_postgresql,postgis,mysql}
    py{310,311}-dj{42,main}-{sqlite,legacy_postgresql,postgresql,postgis,mysql}```

Untested, but should probably fix the problem where you're still running tests with Django 4.1/psycopg3.